BFAIR/INCA/INCA_script_generator.py

```diff
@@ -32,14 +32,18 @@ def limit_to_one_model(self, data_input, model_name_column, model_name):
         """
         Limits the data to values that are assigned to one metabolic model
 
-        Parameters:
-            data_input: Input data file that needs to be processed
-            model_name_column: Column name where model names are defined
-            model_name: Name of the model the data will be limited to
-
-        Returns:
-            data_input: Limited data
-
+        Parameters
+        ----------
+        data_input: pandas.DataFrame
+            Input data file that needs to be processed
+        model_name_column: string
+            Column name where model names are defined
+        model_name: string
+            Name of the model the data will be limited to
+        Returns
```

Comment:
Needs an empty line between "Parameters" section and "Returns" to display properly